test/metabase/api/dashboard_test.clj

```diff
@@ -1436,8 +1437,12 @@
       (mt/with-temp-copy-of-db
         (with-chain-filter-fixtures [{:keys [dashboard param-keys]}]
           (perms/revoke-data-perms! (perms-group/all-users) (mt/id))
-          (is (= "You don't have permissions to do that."
-                 (mt/user-http-request :rasta :get 403 (chain-filter-values-url (:id dashboard) (:category-name param-keys))))))))))
+          ;; HACK: we currently 403 on chain-filter calls that require running a MBQL
+          ;; but 200 on calls that we could just use the cache.
+          ;; It's not ideal and we definitely need to have a consistent behavior
```

Comment:
Is this an actual permissions issue for cached field values?

Reply:
yes : https://metaboat.slack.com/archives/C01NT034X2B/p1650486974835959